### ISSUE TITLE
22.2 RC1: Fast Debug gives warnings for many retail missions, which work fine.

### ISSUE THREAD

**Issue by MoerasGrizzly** [scp-fs2open/fs2open.github.com#4372]
Warning: Ambiguous event name triggers in SM105, 108, loop 1-1, and other missions. However, these missions do not have ambigious event names: The warning can't seem to distinguish upper case from lower case. 

However, this is not a problem in the actual missions, which simply would not work at all if this was an actual problem.

**Comment by Goober5000** [comment]
FRED is case-insensitive across the board; all names with the same letters will match regardless of their case.  If they didn't match, mysterious errors could occur as described in #3221 and elsewhere.

The "ambiguous event name" is a warning, not a true error.  Missions will still run with the same behavior as before, which is to say that the ambiguity is resolved by choosing the first event name that matches.

The warning exists in the first place to alert users of potential unexpected behavior.  If the FREDder gives two events the same name but expects the *second* event to be the trigger, this could lead to confusion.  See #3766.

**Comment by The-E** [comment]
IMHO, the appropriate place for this to emit a warning (and not even a capital-W Warning with the frightening dialogue and log message is when a FREDer is creating new events or renaming existing ones.

**Comment by TRBlount** [comment]
Agreed, this warning should be moved because it technically violates our policy of "Retail shouldn't throw warnings or errors"

**Comment by Goober5000** [comment]
Errors, sure.  Plenty of warnings occur with retail data though.  The "blahblah" dockpoints, duplicate docking bays, LOD table/POF mismatches, to name a few.  SM2-10 has "A file name was expected but no name was supplied".

**Comment by Baezon** [comment]
I'm not sure that's a great policy. Unless we can somehow determine retail data (filenames alone aren't really enough), then retail data is going to totally continue to spew warnings because retail data is genuinely filled with problems. Minor ones, usually, but legitimate ones.

**Comment by MjnMixael** [comment]
What's actionable here? The conversation has been stale for almost 4 years.

**Comment by MoerasGrizzly** [comment]
I'm gonna agree with the "not a bug" marker and close it.
